pkg/rancher-desktop/main/tray.ts

```diff
@@ -99,6 +99,10 @@ export class Tray {
     return os.platform() === 'darwin';
   };
 
+  private isLinux = () => {
+    return os.platform() === 'linux';
+  }
```

Comment:
Lint issues.

I'd say make this a getter, but we should do the same to `isMacOs` too then, and that seems like too much work outside the scope of the PR…

Reply:
Lint issues fixed. I'm open to following up with a refactor later.